internal/provider/provider.go

```diff
@@ -126,6 +126,7 @@ func New() *schema.Provider {
 			"auth0_role_permissions":           role.NewPermissionsResource(),
 			"auth0_rule":                       rule.NewResource(),
 			"auth0_rule_config":                rule.NewConfigResource(),
+			"auth0_scim_configuration":         connection.NewSCIMConfigurationResource(),
```

Comment:
```suggestion
			"auth0_connection_scim_configuration":         connection.NewSCIMConfigurationResource(),
```

This makes it clearer that the scim configuration is something related to the connection resource.